ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLHttpService.java

```diff
@@ -244,8 +249,8 @@ public String url() {
   }
 
   // Facilitate remote health-checks in AWS, inter alia.
```

Comment:
If we are updating the "/" endpoint to no longer be suitable for an AWS health check endpoint we need to update the comment as well.

Reply:
Comment has been updated